models/DataObject/Objectbrick/Definition.php

```diff
@@ -232,7 +232,26 @@ public function save($saveDefinitionFile = true)
         $cd .= 'use Pimcore\Model\DataObject\PreGetValueHookInterface;';
         $cd .= "\n\n";
 
-        $cd .= 'class ' . ucfirst($this->getKey()) . ' extends ' . $extendClass . ' implements \\Pimcore\\Model\\DataObject\\DirtyIndicatorInterface {';
+        $implementsParts = ['\\Pimcore\\Model\\DataObject\\DirtyIndicatorInterface'];
+
+        if ($this->getImplementsInterfaces()) {
+            $customParts = $this->getImplementsInterfaces();
```

Comment:
We should use `DataObject\ClassDefinition\Service::buildImplementsInterfaces` here as well, right?